Solver_interface/doc/Solver_interface/Solver_interface.txt

```diff
@@ -23,9 +23,9 @@ For mixed integer programs (either constrained or unconstrained),
 we provide models using the \ref thirdpartySCIP and \ref thirdpartyGLPK
 libraries.
 
-For linear and quadratic programs, we provide a model using the built-in
-\ref PkgQPSolver "CGAL Linear and Quadratic Programming Solver" and a model using
-the external \ref thirdpartyOSQP library.
+For linear and quadratic programs, CGAl has the built-in
```

Comment:
`CGAl` (typo) should be replaced by `\cgal` or `CGAL`, or `%CGAL`. @sloriot What is recommended?

Reply:
Fixed. I first resolved the conflict in the browser editor and missed that. I am using `\cgal` now. I think it is commonly used in the docs.